src/platform/forms-system/src/js/review/SubmitButtons.jsx

```diff
@@ -146,7 +146,7 @@ export default function SubmitButtons(props) {
       <Row>
         <Column role="alert">{submitMessage}</Column>
       </Row>
-      {preSubmitSection}
+      <preSubmitSection />
```

Comment:
```suggestion
      <PreSubmitSection />
```
Should this be capitalized?